lib/admin-page.php

```diff
@@ -449,6 +449,57 @@ function classicpress_check_can_migrate() {
 	// TODO: Add instructions if WP too old.
 	echo "</td></tr>\n";
 
+	// Check: Conflicting Theme
+	$theme = wp_get_theme();
+	if ( isset( $parameters['themes'] ) && in_array( $theme->stylesheet, (array) $parameters['themes'] ) ) {
+		$preflight_checks['theme'] = false;
+		echo "<tr>\n<td>$icon_preflight_fail</td>\n<td>\n";
+		printf( __(
+			/* translators: active theme name */
+			'It looks like you are using the <strong>%1$s</strong> theme. Unfortuantely it is incompatible with ClassicPress.',
+			'switch-to-classicpress'
+		), $theme->name );
+		echo "<br>\n";
+		_e(
+			'Consider switching to a different theme, perhaps an older core theme, and try again.',
+			'switch-to-classicpress'
+		);
+	} else {
+		$preflight_checks['theme'] = true;
+		echo "<tr>\n<td>$icon_preflight_pass</td>\n<td>\n";
+		printf( __(
+			/* translators: active theme name */
+			'It looks like you are using the <strong>%1$s</strong> theme. We believe it is compatible with ClassicPress.',
+			'switch-to-classicpress'
+		), $theme->name );
+	}
+	echo "</td></tr>\n";
+
+	// Check: Conflicting Plugins
+	$plugins = get_option( 'active_plugins' );
+	if ( isset( $parameters['plugins'] ) && $plugins !== array_diff( $plugins, $parameters['plugins'] ) ) {
+		$preflight_checks['plugins'] = false;
+
+		echo "<tr>\n<td>$icon_preflight_fail</td>\n<td>\n";
+		/* translators: URI to list of conflucting plugins */
+		_e(
+			'We have detected one or more incompatible plugins that prevent migrating your site to ClassicPress. ',
+			'switch-to-classicpress'
+		);
+		echo "<br>\n";
+		printf( __(
+			'Please visit our <a href="%s">Forum</a> for a list of known conflictong plugins.',
+			'switch-to-classicpress'
+		), 'https://docs.classicpress.net/installing-classicpress/#plugin-conflicts' );
```

Comment:
Several issues here:

- This link does not point to the forums as the text indicates.
- Typo: `conflictong`
- The `#plugin-conflicts` part of this link does not work anymore.
- If the list of conflicting plugins comes from a documentation page instead of from the API, then the list will need to be maintained in two different places and it is only a matter of time until they become out of sync.

Reply:
Perhaps on screen reporting of conflicting plugins, see updated PR.